app/locales/sk/exportscreen.json

```diff
@@ -0,0 +1,11 @@
+{
+    "export_para_1":"Môžeš zdielať svoju polohu s kýmkoľvek pomocou tlačidla Zdieľať. Po kliknutí na toto tlačidlo si budeš môcť vybrať s kým a ako chceš svoju polohu zdieľať.",
```

Comment:
"Môžete", not "môžeš", more formal.
"si budete", not "si budeš", more formal.
"ako chcete", not "ako chceš", more formal.